fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -90925,7 +90951,7 @@ creation_date: 2013-05-23T03:45:07Z
 [Term]
 id: FBbt:00110429
 name: adult fruitless aDT-h neuron
-def: "." [FlyBase:FBrf0128543, FlyBase:FBrf0205974, FlyBase:FBrf0211884, FlyBase:FBrf0211926, FlyBase:FBrf0221412, FlyBase:FBrf0221438]
+def: "Any neuron (FBbt:00005106) that is part of some adult fruitless aDT-h lineage clone (FBbt:00110428) and that develops from some neuroblast FLAa3 (FBbt:00047121). These neurons have dendritic arborizations in the subesophageal zone and also project to the superior medial protocerebrum, where they have both presynaptic and postsynaptic terminals (Yu et al., 2010)." [FlyBase:FBrf0128543, FlyBase:FBrf0205974, FlyBase:FBrf0211884, FlyBase:FBrf0211926, FlyBase:FBrf0221412, FlyBase:FBrf0221438]
```

Comment:
Better to just define as:

"Any neuron that develops from neuroblast FLAa3 and expresses fruitless."  
EquivalentTo: neuron that 'develops from' some 'FLAa3 neuroblast' and expresses some 'fru' and 'part of' some 'adult brain'
subClassOf: 'part of' some 'adult fruitless aDT-h lineage clone'
...

This is the preferred pattern for all lineage clone & their neurons. I suspect there may be some work to do to review and improve other fru neuron defs.

Reply:
Yeah a lot of similar fruitless neurons just have "." definitions and have the lineage clone as part of the equivalent class statement.